src/pages/waypoint/downloads/index.tsx

```diff
@@ -28,10 +28,6 @@ const WaypointDownloadsSidecarContent = () => {
 }
 
 const WaypointDowloadsPage = (): ReactElement => {
-  const backToLink = {
-    text: 'Back to Waypoint',
-    url: '/waypoint',
-  }
```

Comment:
What's happening here 💭

Reply:
Ah, this is temporary! It's needed in order to show that the placeholder back to link's focus still works. I couldn't find a different page where the placeholder was rendering. Will be changed back before launching the page 😊